python/doc/user_manual/functions.rst

```diff
@@ -83,7 +83,6 @@ Refer to :ref:`field_function`.
     OpenTURNSPythonFieldFunction
     OpenTURNSPythonFieldToPointFunction
     OpenTURNSPythonPointToFieldFunction
-    P1LagrangeInterpolation
```

Comment:
it must not be removed, but you can move it below near P1LagrangeEvaluation

Reply:
I let it where it was...